pynecone/state.py

```diff
@@ -105,18 +99,10 @@ def __init__(self, *args, parent_state: Optional[State] = None, **kwargs):
         self._init_mutable_fields()
 
         # Initialize computed vars dependencies.
-        self.computed_var_dependencies = defaultdict(set)
-        for cvar in self.computed_vars:
-            self.tracked_vars = set()
-
-            # Enable tracking and get the computed var.
-            self.track_vars = True
-            self.__getattribute__(cvar)
-            self.track_vars = False
-
+        for cvar_name, cvar in self.computed_vars.items():
```

Comment:
I think we may need to keep the line `self.computed_var_dependencies = defaultdict(set)` - because of Python weirdness when we set the initial value to a mutable type like list/dict it becomes shared by all the subclasses. We ran into this issue a couple times

Reply:
_that_ probably explains the weirdness i was hitting w.r.t. accessing that value in a class context vs an instance context. will rework and see if i can add a test for this.

Reply:
@picklelo i moved the `defaultdict` assignment back into `__init__`, so that each instance gets a copy, rather than sharing the class copy...

However to make it work, I had to also move the `_init_mutable_values` call _after_ `computed_var_dependencies` is assigned, because `_init_mutable_vars` might call `__setattr__`, which will call `mark_dirty`, which attempts to look up any dirty vars in the `computed_var_dependencies`, which won't work, unless it's been assigned as a default dict... turns out _that_ was the problem I was hitting before.

After moving the `_init_mutable_values` call, there's no problem with assigning `computed_var_dependencies` in `__init__`.